examples/python/garn.ts

```diff
@@ -0,0 +1,7 @@
+import * as garn from "http://localhost:8777/mod.ts";
+import * as pkgs from "http://localhost:8777/nixpkgs.ts";
+
+export const someproject = garn.python.mkPythonProject({
+  src: ".",
+  pythonInterpreter: garn.python.interpreters.python310,
```

Comment:
Elsewhere we have been doing only strings (e.g. "python-3.1.0", or just "3.1.0"), with the type being the union of the allowed strings. 

This makes it a bit more discoverable, at the cost of being a bit more work for us to generate those. But overall I think consistency is the main gain.

Reply:
I did implement it as strings first, but then thought, it would be more flexible by using an expression type instead. That way, the user could easily override the python interpreter in arbitrary ways by passing an expression.
But I understand the consistency concern. I can go for the strings for now, if that's preferred.